styles/styles.css

```diff
@@ -48,6 +56,11 @@
   src: local('Arial');
 }
 
+@font-face {
```

Comment:
Probably we should load this delayed with fallback

Reply:
moved to fonts.css for delayed loading